configs/components/_base-ruby.rb

```diff
@@ -59,6 +59,7 @@
     end
     pkg.environment 'CC', 'clang -target arm64-apple-macos11' if platform.name =~ /osx-11/
     pkg.environment 'CC', 'clang -target arm64-apple-macos12' if platform.name =~ /osx-12/
+    pkg.environment 'CC', 'clang -target arm64-apple-macos13' if platform.name =~ /osx-13/
```

Comment:
My understanding is that we only need to specify `--target` in cases where we're cross compiling. If the `host` and `target` are the same OS/arch, then `--target` shouldn't be needed (?) If that's true, then I'm not sure we need to set `CC` at all (?) Or does ruby default to gcc on macOS?

```suggestion
    if platform.is_cross_compiled?
       ...
       pkg.environment 'CC', 'clang -target arm64-apple-macos11' if platform.name =~ /osx-11/
       pkg.environment 'CC', 'clang -target arm64-apple-macos12' if platform.name =~ /osx-12/
    else
       pkg.environment 'CC', 'clang'
    end    
```